components/FlexibleHeader/tests/unit/FlexibleHeaderHeightTests.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+import MaterialComponents.MaterialFlexibleHeader
+
+class FlexibleHeaderHeightTests: XCTestCase {
+
+  func testDefaults() {
+    // Given
+    let flexibleHeaderView = MDCFlexibleHeaderView()
+    flexibleHeaderView.minMaxHeightIncludesSafeArea = true
+
+    // Then
+    XCTAssertEqual(flexibleHeaderView.minimumHeight, 76, accuracy: 0.0001)
```

Comment:
Should we be mocking the safe area size somehow for this test?  It seems weird to just assume it's 20 when it can depend on the superview's safe area.

Reply:
I've faked out the safe area logic now.